recipes-graphics/mesa/mesa_%.bbappend

```diff
@@ -14,4 +14,20 @@ do_install_append_hikey() {
     rm -f ${D}/${libdir}/libwayland-egl.*
 }
 
+do_install_append_hikey-32() {
```

Comment:
And likely needs to be converted to a MACHINE_FEATURE = "malibinary", otherwise we'll need to duplicate it for each and every mali machine.